README.md

```diff
@@ -1,118 +1,106 @@
-# Jenkins Folder Properties Plugin
+The folder-properties plugin is a Jenkins plugin which allows users with config permission to define
+properties for a folder which can then be used by any jobs contained
+within it or in any of its sub-folders. 
 
-A Jenkins plugin which allows users with config permission to define properties for a folder which can then be used by
-any jobs contained within it or any of its sub-folders.
+# About this Plugin
 
-The aim here is to remove the need to specify the same properties over and over again for all the jobs inside a folder.
+The aim here is to remove the need to specify the same properties over
+and over again for all the jobs inside a folder.
 
-In structures where two or more folders are nested, any property defined for a folder will be overridden by any other property of the same name defined by one of its sub-folders.
+## How to use?
 
-## Develop
-
-For ease of development I've added a `logging.properties` file that can be used by specifying it as an option to the
-Maven `hpi:run` target like so:
-
-    mvn hpi:run -Djava.util.logging.config.file=logging.properties
-
-## Package
-
-Since this is my first version of this and I'm still missing the tests...
-
-    mvn package -DskipTests=true
-
-## Use
+To configure, just create a
+[folder](https://plugins.jenkins.io/cloudbees-folder/),
+go to its configuration page and add as many properties as you need
+under the `Folder Properties` section.
 
 In structures where two or more folders are nested, any property defined for a folder will be overridden by any other
 property of the same name defined by one of its sub-folders.
 
-<img src="./pics/folder-properties-1.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/>
+![](docs/images/folder-properties-config.png)
 
-### Freestyle Jobs
 
-Freestyle jobs must opt into the `Folder Properties` build wrapper from the `Build Environment` section of their
-configuration page in order to be able to access these properties as they would any other environment variable.
+## Freestyle Jobs
 
-<img src="./pics/freestyle-build-env.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+Freestyle jobs must opt into the `Folder Properties` build wrapper from
+the `Build Environment` section of their configuration page in order to
+be able to access these properties as they would any other environment
+variable.
 
-Only then will they inherit properties defined by their parent or ancestor folders —e.g. Running `echo $FOO` in a Shell build step:
+![](docs/images/folder-properties-freestyle-config.png)
 
-<img src="./pics/freestyle-example-1.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+Only then will they inherit properties defined by their parent or ancestor folders —e.g. Running `echo $FOO` in a Shell build step :
```

Comment:
General guidance from writers is to avoid the Latin abbreviation `e.g.` and replace it with literal 'for example'.  Optional
```suggestion
Only then will they inherit properties defined by their parent or ancestor folders — for example, running `echo $FOO` in a Shell build step :
```